src/generate-services.ts

```diff
@@ -121,11 +121,11 @@ function generateRegularRpcMethod(
   const maybeCtx = options.context ? "ctx," : "";
 
   let encode = code`${rawInputType}.encode(request).finish()`;
-  let decode = code`data => ${outputType}.decode(new ${Reader}(data))`;
+  let decode = code`data => ${rawOutputType}.decode(new ${Reader}(data))`;
 
-  if (options.useDate && rawOutputType.toString().includes("Timestamp")) {
-    decode = code`data => ${utils.fromTimestamp}(${rawOutputType}.decode(new ${Reader}(data)))`;
-  }
+  // if (options.useDate && rawOutputType.toString().includes("Timestamp")) {
+  //   decode = code`data => ${utils.fromTimestamp}(${rawOutputType}.decode(new ${Reader}(data)))`;
+  // }
```

Comment:
I assume this is related to https://github.com/stephenh/ts-proto/pull/689 , just checking that you wanted to include commenting out this code in this PR?